examples/posenet/camera.js

```diff
@@ -0,0 +1,115 @@
+async function main(){
+  const video = document.getElementById('video');
+  const canvas = document.getElementById('canvas');
+  const ctx = canvas.getContext('2d');
+  const scaleCanvas = document.getElementById('canvas_2');
+  const scaleCtx = scaleCanvas.getContext('2d');
+  const util = new Utils();
+  const videoWidth = 500;
+  const videoHeight = 500;
+  const inputSize = [1, videoWidth, videoHeight, 3];
+  const isMultiple = guiState.algorithm;
+  let streaming  = false;	
+  if (!navigator.mediaDevices || !navigator.mediaDevices.getUserMedia) {
+    throw new Error(
+      'Browser API navigator.mediaDevices.getUserMedia not available');
+  }
+  let stats = new Stats();
+  stats.dom.style.cssText = 'position:fixed;top:60px;right:10px;cursor:pointer;opacity:0.9;z-index:10000';
```

Comment:
Put the stats gui to left. Change `right` to `left`. See https://github.com/intel/webml-polyfill/blob/master/examples/mobilenet/camera.js#L54